tests/src/main/java/org/hibernate/beanvalidation/tck/tests/messageinterpolation/ExpressionLanguageMessageInterpolationTest.java

```diff
@@ -241,4 +253,33 @@ public boolean isValid(Continent continent, ConstraintValidatorContext constrain
 			}
 		}
 	}
+
+	private static class TestWrapper {
+
+		private final TestLogic testLogic;
+		private final Locale originalLocale;
+		private final Locale testLocale;
+
+		public TestWrapper(TestLogic testLogic, Locale testLocale) {
+			this.testLogic = testLogic;
+			this.originalLocale = Locale.getDefault();
+			this.testLocale = testLocale;
+		}
+
+		public void doTest() {
+			Locale.setDefault( testLocale );
+			testLogic.test();
+			Locale.setDefault( originalLocale );
+		}
+
+		public static void test(TestLogic testLogic) {
+			new TestWrapper( testLogic, Locale.ENGLISH ).doTest();
```

Comment:
@marko-bekhta we discussed it with Gunnar and we think it's better to simply disable checkstyle for these imports in this test as the usage is legitimate here (see https://github.com/beanvalidation/beanvalidation-tck/pull/91#issuecomment-287716857 ).

The test will be more readable this way.